src/covid_model_seiir_pipeline/model_runner.py

```diff
@@ -41,3 +43,33 @@ def forecast(model_specs, init_cond, times, betas, thetas=None, dt=0.1):
         """
         forecaster = ODERunner(model_specs, init_cond, dt=dt)
         return forecaster.get_solution(times, beta=betas, theta=thetas)
+
+
+# FIXME: The only "modeling" code shared between the stages.  Where to put it?
```

Comment:
`from covid_model_seiir_pipeline.math import compute_beta_hat`?

Reply:
yeah, something like that.